src/fallback.ts

```diff
@@ -365,16 +374,29 @@ export function lro(options: GrpcClientOptions) {
 export function createApiCall(
   func: Promise<GRPCCall> | GRPCCall,
   settings: gax.CallSettings,
-  descriptor?: Descriptor
+  descriptor?: Descriptor,
+  fallback?: boolean | 'proto' | 'rest'
 ): GaxCall {
   if (
+    (!fallback || fallback === 'rest') &&
     descriptor &&
     'streaming' in descriptor &&
     (descriptor as StreamDescriptor).type !== StreamType.SERVER_STREAMING
   ) {
     return () => {
       throw new Error(
-        'The gRPC-fallback client library (e.g. browser version of the library) currently does not support client-streaming or bidi-stream calls.'
+        'The REST transport currently does not support client-streaming or bidi-stream calls.'
+      );
+    };
+  }
+  if (
```

Comment:
I don't see these errors being tested explicitly.

Reply:
The browser use case tests it in `test/browser-test/test.endtoend.ts`, lines 86-92.
It seems the REGAPIC use case (the first `throw`) is indeed not tested, I added tests for all these cases to the test application (`test/test-application/src/index.ts`, functions `testExpandThrows`, `testCollectThrows`, `testChatThrows`). Thank you for noticing it!